cloudinary/cloudinary_wordpress
# Convert gif to video if f_mp4 is present

Test instructions:

- Upload an animated gif
- Under global image transformations make sure to add `f_mp4`
- Use the animated gif in a post
- Observe that the gif will be served as a video instead, fully complying with the video settings
